src/htmlminifier.js

```diff
@@ -665,7 +665,11 @@ function processOptions(values) {
           return prefix + quote + options.minifyURLs(url) + quote + suffix;
         });
         try {
-          return unwrapCSS(new CleanCSS(value).minify(wrapCSS(text, type)).styles, type);
+          var cleanCssOutput = new CleanCSS(value).minify(wrapCSS(text, type));
+          if (cleanCssOutput.errors.length > 0) {
+            throw new Error(cleanCssOutput.errors[0]);
+          }
+          return unwrapCSS(cleanCssOutput.styles, type);
```

Comment:
Does `clean-css` throw anymore?

If not, let's get rid of the whole try-catch, plus that way we get to log all the errors not just the first one in the list.

Reply:
Only for unhandled exceptions, but most of them should be handled.
Done in https://github.com/kangax/html-minifier/pull/955/commits/56e40e4f726ec2e7599377680e059a2ffdb06ffa @alexlamsl